src/style.css

```
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.header__link {
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: inline-block;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  padding: 23px 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:hover,
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border-radius: 8px;
+  background: #00acdc;
+  width: 37px;
```

```suggestion
  width: 100%;
```
Try to not hardcode the width, it should depend on the parent element